design-editor/src/pane/design-editor-element.js

```diff
@@ -618,12 +618,18 @@ class DesignEditor extends DressElement {
 	 * @private element
 	 */
 	_refreshTAUWidget(id) {
-		const element = this._getElementById(id),
-			tau = this._$iframe[0].contentWindow && this._$iframe[0].contentWindow.tau;
+		const element = this._getElementById(id)[0],
```

Comment:
I prefer .get() method to retrieve the DOM element.
https://api.jquery.com/get/#get-index

Reply:
I've changed it.